docs/en/reference/information_schema/pipe_files.md

```diff
@@ -0,0 +1,24 @@
+---
+displayed_sidebar: "English"
+---
+
+# pipe_files
+
+`pipe_files` provides the status of the data files to be loaded via a specified pipe. This view is supported from StarRocks v3.2 onwards.
+
+The following fields are provided in `pipe_files`:
+
+| **Field**        | **Description**                                              |
+| ---------------- | ------------------------------------------------------------ |
+| DATABASE_NAME    | The name of the database in which the pipe is stored.        |
+| PIPE_ID          | The unique ID of the pipe.                                   |
+| PIPE_NAME        | The name of the pipe.                                        |
+| FILE_NAME        | The number of the data file.                                 |
+| FILE_VERSION     | The digest of the data file.                                 |
+| FILE_SIZE        | The size of the data file. Unit: bytes.                      |
+| LAST_MODIFIED    | The last time when the data file was modified. Format: `yyyy-MM-dd HH:mm:ss`. Example: `2023-07-24 14:58:58`. |
+| LOAD_STATE       | The load status of the data file. Valid values: `UNLOADED`, `LOADING`, `FINISHED`, and `ERROR`. |
+| STAGED_TIME      | The date and time when the data file was first recorded by the pipe. Format: `yyyy-MM-dd HH:mm:ss`. Example: `2023-07-24 14:58:58`. |
+| START_LOAD_TIME  | The date and time when the loading of the data file began. Format: `yyyy-MM-dd HH:mm:ss`. Example: `2023-07-24 14:58:58`. |
+| FINISH_LOAD_TIME | The date and time when the loading of the data file finishes. Format: `yyyy-MM-dd HH:mm:ss`. Example: `2023-07-24 14:58:58`. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/38cdbf03-971a-4808-8a8d-bfb2909382bd)
-> finished
(this one also should blame on me.)